ee/disclaim/run_disclaimed_darwin.go

```diff
@@ -98,6 +98,13 @@ var allowedCmdGenerators = map[string]allowedCmdGenerator{
 		},
 		generate: allowedcmd.Zscli,
 	},
+	"microsoft_defender_atp": {
+		allowedOpts: map[string]struct{}{
+			"health":        {},
+			"--output json": {},
```

Comment:
If this works like I expect it to, this should probably be 2 array entries, or `"--output": "json"` Two array entries is probably cleaner.